ydb/core/kqp/session_actor/kqp_session_actor.cpp

```diff
@@ -394,7 +397,6 @@ class TKqpSessionActor : public TActorBootstrapped<TKqpSessionActor> {
         YQL_ENSURE(QueryState->GetDatabase() == Settings.Database,
                 "Wrong database, expected:" << Settings.Database << ", got: " << QueryState->GetDatabase());
 
-        QueryState->EnsureAction();
```

Comment:
Is this not needed anymore?

Reply:
This ensure is here https://github.com/ydb-platform/ydb/pull/6049/files#diff-d9749fd0cd55fd4d28688918a2d479a043a4c2d1f56a6124150ff741c0c640a1R77